src/PayU/Alu/Client.php

```diff
@@ -177,20 +62,26 @@ public function pay(Request $request, HTTPClient $httpClient = null, HashService
             $httpClient = new HTTPClient();
```

Comment:
Let's move instantiation of the default HTTPClient and HashService to the factory class, if null.

Reply:
Done

Reply:
Also, instead of creating an instance of `PayU\Alu\HTTPClient`, I think we can create an instance of `PayU\PaymentsApi\AluV3\Services\HTTPClient`

Reply:
Good point -> I've changed it